sync-todo/v2/client/maui/RealmTodo/ViewModels/EditItemViewModel.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using RealmTodo.Models;
+using RealmTodo.Services;
+
+namespace RealmTodo.ViewModels
+{
+    public partial class EditItemViewModel : BaseViewModel, IQueryAttributable
+    {
+        [ObservableProperty]
+        private Item newOrChangedItem;
+
+        [ObservableProperty]
+        private string dialogTitle;
+
+        private Realms.Realm realm;
+        private string currentUserId;
```

Comment:
We are not using this anywhere else than "ApplyQueryAttributes", this can be removed and we can just use a local variable there or even use "RealmService.CurrentUser.Id" when needed

Reply:
I did the same with "realm" because it is also only used once in the same method.